### PR TITLE
OKTA-256102 Fix Kosovo country code

### PR DESCRIPTION
**Pull request by haoyuli-okta**
Fix Kosovo country code for SMS/Voice

### CODE REVIEW

**Comment by haishengwu-okta** [comment]
how did you update this file?

**Comment by haoyuli-okta** [comment]
I added the country code directly to this file 'XK':'383', and verified it worked locally

**Comment by haishengwu-okta** [comment]
you probably shall change 
https://github.com/okta/okta-signin-widget/blob/master/buildtools/commands/generate-phone-codes.js#L39
then run `yarn generate-phone-codes` which would update the file that you manually updated.

**Comment by haoyuli-okta** [comment]
Tested with Okta-Core on end-user reset SMS/Voice flow, also tested with enrollment flow and authentication flow

File: src/util/countryCallingCodes.js [hunk 1]
```diff
@@ -1,13 +1,253 @@
-/*!
```
Comment (haishengwu-okta) [inline src/util/countryCallingCodes.js]: the scripts still have defect (didn't add copyright notes). would you mind add copyright manually after file has been created?

Comment (mauriciocastillosilva-okta) [reply src/util/countryCallingCodes.js]: +1 to add back copyright notes

Comment (haoyuli-okta) [reply src/util/countryCallingCodes.js]: fixed

**Pull request merged**